marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/flow/job-query-lib.sjs

```diff
@@ -141,6 +141,26 @@ function buildFacetQueries(whereClause) {
   return queries;
 }
 
+function getMatchingPropertyValues(facetValuesSearchQuery) {
+  const facetName = facetValuesSearchQuery.facetName;
+  const searchTerm = facetValuesSearchQuery.searchTerm;
+  const limit = facetValuesSearchQuery.limit ? facetValuesSearchQuery.limit : 10 ;
```

Comment:
I don't see a test for limit; you could do this pretty easily by having 2 different values, and verify you get both back without a limit, and then submit a limit of 1 to verify you get 1 back.